src/main/java/de/tum/cit/aet/artemis/iris/service/settings/IrisSubSettingsService.java

```diff
@@ -459,7 +460,8 @@ private <T extends IrisSubSettings> boolean getCombinedEnabled(List<IrisSettings
                 return false;
             }
             var settings = subSettingsFunction.apply(irisSettings);
-            if (settings == null || !settings.isEnabled()) {
+            // enabled/disabled should no longer be toggled using global settings
+            if (settings == null || (!settings.isEnabled() && !(irisSettings instanceof IrisGlobalSettings))) {
```

Comment:
That's not correct from my POV. That would render the switches in the global settings useless.